### PR TITLE
Dashboard page

### PR DESCRIPTION
**Pull request by JanaKhanji**
I had to change some stuff in the components.
also please check the data.js

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  },
+];
+
+export const trackers = [
+  {
+    id: 1,
+    title: 'brush my teeth',
+    duration: 14,
+    StartDate: 'Sat Feb 20 2021',
+    ongoing: true,
```
Comment (JanaKhanji) [inline src/data.js]: make it completed instead of ongoing

File: src/Components/Dashboard/Tracker/Tracker.js [hunk 2]
```diff
@@ -2,12 +2,18 @@
 import Addnew from '../../../assets/images/Addnew.png';
 
 function Tracker({ tracker }) {
+  const today = null;
+  if (tracker) {
+    today = tracker.track.filter(
```
Comment (JanaKhanji) [inline src/Components/Dashboard/Tracker/Tracker.js:7]: const today !!!

**Review by osamaakb — CHANGES_REQUESTED** [review]
Good job @JanaKhanji , this looks really nice, 2 quick comments I have, 
Then Navbar background doesn't match the rest of the page, see the image below

image

And the page has some vertical scrolling, can we have it not scrollable?

**Comment by JanaKhanji** [comment]
I don't think we can have it without the vertical scrolling because the components in the first column are large and they didn't fit.
also for the light grey background, the navbar is a global component so it is in the app next to the dashboard page, and the background should be applied to the whole app style because it is the color of all the pages so I didn't do it here

**Comment by JanaKhanji** [comment]
should I remove the weather component to prevent the scrolling?

**Comment by osamaakb** [comment]
> should I remove the weather component to prevent the scrolling?

if that will fix the problem, it's better to remove it then

**Comment by osamaakb** [comment]
> I don't think we can have it without the vertical scrolling because the components in the first column are large and they didn't fit.
> also for the light grey background, the navbar is a global component so it is in the app next to the dashboard page, and the background should be applied to the whole app style because it is the color of all the pages so I didn't do it here

can you then make it's background transparent or something to take the background of the app, cause it doesn't look really good like this

**Comment by JanaKhanji** [comment]
> > should I remove the weather component to prevent the scrolling?
> 
> if that will fix the problem, it's better to remove it then

**Pull request closed without merging**